pristine-zen-garden.html

```diff
@@ -10,7 +10,6 @@
     <meta name="description" content="A demonstration of what can be accomplished visually through CSS-based design.">
     <meta name="robots" content="all">
 </head>
-
 <body id="css-zen-garden">
     <div class="page-wrapper">
         <section class="intro" id="zen-intro">
```

Comment:
Section and header need their contents indented (lines 15 - 18)

Reply:
contents of p tags could be made multiline so that less sideways scrolling is needed